distro/microfrontends.json

```diff
@@ -0,0 +1,28 @@
+{
+  "microfrontends": {
+    "@pih/esm-refapp-navbar-app": "latest",
+    "@pih/esm-referrals-queue-app": "latest",
+    "@openmrs/esm-home-app": "next",
```

Comment:
Out of curiosity, what does "next" mean in terms of version?

Reply:
It's a tag. Generally (and in our case) it's used for the latest non-release version. So e.g. `latest` might be 1.1.0 but `next` might be `1.2.0-beta3`.

Reply:
Ah, so it's basically the equivalent of "snapshot" from maven... sounds good. And probably a better word that "snapshot"... I hated "snapshot" at first but got used to it.